versions/1/changelog.yml

```
@@ -9,6 +9,9 @@
     link: https://github.com/elastic/package-spec/pull/265
 - version: 1.4.0
   changes:
+  - description: Updates the ml_model file format pattern for validation.
+    type: enhancement
```

I'd suggest to consider this a breaking change, as files matching the previous pattern won't match the new one.

I'd also rephrase the changelog entry being more explicit about the update in the pattern. Maybe something like this:
```suggestion
  - description: ML model file name matches now the id of the model.
    type: breaking-change
```

---

This isn't a breaking change as the only packages that will now use this have not been merged into integrations yet. No packages have used it so far.

---

Made the suggested changelog changes in 9b30841d1905134ebfb81f5d24833e6e53446732